api/api/utils/check_dead_links/__init__.py

```diff
@@ -32,9 +33,11 @@ def _get_expiry(status, default):
     return config(f"LINK_VALIDATION_CACHE_EXPIRY__{status}", default=default, cast=int)
 
 
-async def _head(url: str, session: aiohttp.ClientSession) -> tuple[str, int]:
+async def _head(url: str, **kwargs) -> tuple[str, int]:
```

Comment:
Changes in this module are just to support the reusable aiohttp session. We should have been doing that already anyway.